### ISSUE TITLE
can I use this code to generate a summary from a new source text

### ISSUE THREAD

**Issue by hanyegirl** [iwangjian/textsum-gan#10]
i want to generate a summary ,and the source text is not in the text.bin ,just an ordinary English text,what can i do for this task ,thank you for your any help

**Comment by iwangjian** [comment]
The processing from text data to ```.bin``` file is referenced by https://github.com/abisee/cnn-dailymail, you can refer to this code to process your own data.

**Comment by hanyegirl** [comment]
> The processing from text data to .bin file is referenced by https://github.com/abisee/cnn-dailymail, you can refer to this code to process your own data.
then use this 
python3 main.py --mode=decode --data_path=./data/test.bin --vocab_path=./data/vocab --log_root=./log --single_pass=True 
to generate the relative summary?

**Comment by iwangjian** [comment]
Yes, but firstly you need to train a model using your own data following the instructions in ```README.md```, and then use the trained model to generate the relative summary.

**Comment by hanyegirl** [comment]
thank you ,your advice is very important to me

**Comment by HunterYCG** [comment]
May I ask, after you use the new data set to train the model, what is the test result? At present, I am also doing some related work, so I would like to consult you.

**Comment by iwangjian** [comment]
If you use a new dataset to train the model, the test input are texts (or articles) in your test set, the test output are corresponding generated summaries.

**Comment by HunterYCG** [comment]
Thank you very much for your reply.
